pkg/io/storage/storage.go

```diff
@@ -0,0 +1,90 @@
+/*
+Package storage provides functions for storing and retrieving data.
+from json file
+*/
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+type fileVault struct {
+	storage map[string][]byte
+	path    string
+	file    *os.File
```

Comment:
I suppose that with the scheme where data are saved and written to a file on every action there is no need to store a reference on file descriptor since it will have no use. Method `Close()` in those realities is unnecessary because no real job will be done and could be also removed.